components/servo/Cargo.lock

```diff
@@ -183,6 +183,11 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "bitflags"
+version = "0.4.0"
```

Comment:
It needs https://github.com/nix-rust/nix/pull/431. I don't think it is a problem to include 2 versions of bitflags given how small that crate is. It was already an exception in the past anyway.